generic/piranha/src/cleanup_rules/java/rules.toml

```diff
@@ -380,7 +380,7 @@ queries = [
 (
 ((assignment_expression
                     left: (_) @a.lhs
-                    right: (_) @a.rhs) @a)
+                    right: (_) @a.rhs) @assignment)
```

Comment:
bit surprised with this change. The other variables are not renamed, yet this works?

Reply:
oh yea. as of now I have no logic that tries to create any relationship between tags based on their name. IT could be a interesting direction to explore though.